.travis.yml

```diff
@@ -5,25 +5,28 @@ python:
   - "3.4"
   - "3.5"
 
+sudo: false
+
 services:
   - elasticsearch
-  
+
 install:
   - python setup.py install
   - pip install pyflakes
   - pip install pep8
   - pip install docutils
   - pip install nose
-  - pip install coveralls
+  - if [[ $TRAVIS_PYTHON_VERSION == '3.5' && $PYTHONASYNCIODEBUG == '0' ]]; then pip install elasticsearch coverage coveralls; fi
```

Comment:
if asyncio debug is disabled `PYTHONASYNCIO` is equal to empty string not `0`.
See the latest `travis.yml` version.